### PR TITLE
feat add elasticsearch stack

### PR DESCRIPTION
**Pull request by guorex**

### CODE REVIEW

**Comment by pahud** [comment]
Hi @guorex 

You probably need run `yarn upgrade` in this PR to update the yarn.lock file.

**Review by pahud — COMMENTED** [review]
LGTM

File: src/analytics.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          principals: [
+            new iam.AnyPrincipal
+          ],
+          resources: [`arn:aws:es:${region}:${account}:domain/${esDomainName}/*`
+          ]
+        });
+
+        //Create Elasticsearch service
+        const esDomain = new es.Domain(this, "ElasticsearchDomain", {
+          version: es.ElasticsearchVersion.V7_4,
```
[the remark this answers is not in the document]
Comment (guorex) [reply src/analytics.ts:38]: Ok, thanks for your suggestion

File: package.json [hunk 2]
```diff
@@ -189,4 +191,4 @@
     "twitter": "pahudnet",
     "announce": false
   }
-}
\ No newline at end of file
+}
```
Comment (pahud) [inline package.json:194]: `yarn upgrade` will generate the trailing newline. You may run `npx projen` again to fix it.

**Review by pahud — CHANGES_REQUESTED** [review]

**Review by pahud — APPROVED** [review]
Ship it!

**Pull request merged**